bolt.yaml

```diff
@@ -0,0 +1,20 @@
+# puppetdb:
+#   server_urls: ["https://foo.example.com:8081"]
+#   cacert: /path/to/ca.pem
+#   token: ~/.puppetlabs/token
+# pcp:
+#   service-url: "https://foo.example.com:8143"
+#   cacert: /path/to/ca.pem
+#   task-environment: production
+# ssh:
+#   host-key-check: false
+#   private-key: ~/.ssh/bolt_id
+#   user: foo 
+#   interpreters:
+#      rb: /home/foo/.rbenv/versions/2.5.1/bin/ruby
+# log:
+#   console:
```

Comment:
Configuring the console log level is a strange use case given BOLT-901, so we might want to omit that as well